Exploration/README.md

```diff
@@ -0,0 +1,23 @@
+# 🧗 Exploration
+
+Ready to flex your mental muscles? Check out
+these learning resources, approved by Launch
```

Comment:
this line width seems much shorter than the other files?

Reply:
Fixing.